examples/webpack/src/utils/getMorphologyData.js

```diff
@@ -0,0 +1,21 @@
+let morphologyData = null;
+
+function loadLocalMorphologyData() {
```

Comment:
Although it is pretty straightforward what these functions do, it would be nice to add some JSDoc just in case 😉.
I will add this myself, and move it to code review done afterwards.